Cargo.toml

```diff
@@ -36,11 +37,16 @@ tempfile = "3.8"
 arrow = { version = "50.0", default-features = false, features = ["ffi"] }
 
 [workspace]
-members = ["gdal-sys"]
+members = ["gdal-src", "gdal-sys"]
 
 # docs.rs-specific configuration
 [package.metadata.docs.rs]
 # include `array` feature in documentation
 features = ["array"]
 # define attribute `docsrs` for feature badges
 rustdoc-args = ["--cfg", "docsrs"]
+
+
+[patch.crates-io]
+proj-sys = { git = "https://github.com/GiGainfosystems/proj", rev = "44a503869709ea793a5f8452bfb9e2a99b7160fc" }
+netcdf-src = { git = "https://github.com/GiGainfosystems/netcdf", rev = "292309979f533f71d542f4fcf8fdac8830f0e22b" }
```

Comment:
I've opened https://github.com/Unidata/netcdf-c/pull/2906 to fix the netcdf duplicated symbol issue. Let's see if upstream accepts that PR, after that we need to merge the fix into the rust bindings crate. (Which is under the georust github organization)

Reply:
The upstream PR to netcdf itself is accepted and merged :tada:

I've opened one against the rust bindings here to include the fix there as well: https://github.com/georust/gdal/pull/517

Reply:
I've pushed https://github.com/georust/gdal/pull/517/commits/50dd40516c85fbf26d46632fe3f51c1c61e42141 which pulls in netcdf-src 0.3.3 which contains the relevant fix.